mobile/src/stateMachine/views/password/passwordBase.tsx

```diff
@@ -129,7 +136,7 @@ export abstract class PasswordBase extends ViewState {
                         { height: containerHeight, paddingTop: containerPadding },
                     ]}>
                         <View style={[this.baseStyles.textBlock, styles.textBlock, keyboard?.isOpened ? { position: 'absolute', top: containerPadding } : null]}>
-                <Text style={[scaleDownTitle ? this.textStyles.h3 : this.textStyles.h1, styles.title]}>{this.title}</Text>
+                            <Text style={[scaleDownTitle ? this.textStyles.h3 : this.textStyles.h1, styles.title]}>{this.title}</Text>
```

Comment:
lint

Reply:
this code was not linted before, so I linted and it has gotten some extra spaces to it.